src/DependencyInjection/Compiler/DoctrineOrmMappingsPass.php

```diff
@@ -47,17 +52,27 @@ public function __construct(Definition|Reference $driver, array $namespaces, arr
     }
 
     /**
-     * @param string[]     $namespaces        Hashmap of directory path to namespace.
-     * @param string[]     $managerParameters List of parameters that could which object manager name
-     *                                        your bundle uses. This compiler pass will automatically
-     *                                        append the parameter name for the default entity manager
-     *                                        to this list.
-     * @param string|false $enabledParameter  Service container parameter that must be present to
-     *                                        enable the mapping. Set to false to not do any check,
-     *                                        optional.
+     * @param string[]      $namespaces          Hashmap of directory path to namespace.
+     * @param string[]      $managerParameters   List of parameters that could which object manager name
+     *                                           your bundle uses. This compiler pass will automatically
+     *                                           append the parameter name for the default entity manager
+     *                                           to this list.
+     * @param string|false  $enabledParameter    Service container parameter that must be present to
+     *                                           enable the mapping. Set to false to not do any check,
+     *                                           optional.
+     * @param string[]|bool $enableXsdValidation This should never be an array, but is is due to BC reasons.
```

Comment:
Only the expected type should be documented, for static analysis.
```suggestion
     * @param bool $enableXsdValidation
```

Reply:
PHPStan complains if I just use bool here, that's why I did it that way. (See the PHPStan error)
Not sure if you can just ignore that error somehow or if it should be PHPStan compliant with the message.

Reply:
@GromNaN I added a phpstan ignore directive to suppress this now which seems to make the tests run fine. I hope that works with the coding standards because I don't see how that's solved otherwise